internal/server/server.go

```diff
@@ -97,6 +97,7 @@ func EchoEngine(ctrl Controller) *echo.Echo {
 	restricted.POST("/auth/sign_out", auth.Logout)
 	restricted.POST("/auth/update", auth.Update)
 	restricted.POST("/auth/change_pw", auth.UpdatePassword)
+	v1restricted.PUT("/users/:id/attributes/credentials", auth.UpdatePassword)
```

Comment:
It returns a `500 Internal Server Error` when the new email is used by antother user.
`could not persist user: could not save the model: already exists`

Reply:
Thanks for implementing an additional check!  :+1: